### PR TITLE
update retroplayer add-ons and emulation packages

### PR DESCRIPTION
**Pull request by lrusak**
built and pushed for RPi2 and Generic

libretro-bsnes-mercury-performance didn't want to build so I've disabled it for now

### CODE REVIEW

File: packages/mediacenter/kodi-binary-addons/game.libretro.craft/package.mk [hunk 1]
```diff
@@ -17,9 +17,9 @@
 ################################################################################
 
 PKG_NAME="game.libretro.craft"
-PKG_VERSION="0201962"
+PKG_VERSION="22c1e8a"
 PKG_SHA256="e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```
Comment (CvH) [inline packages/mediacenter/kodi-binary-addons/game.libretro.craft/package.mk]: Missing sha256?

File: packages/mediacenter/kodi-binary-addons/game.libretro.dosbox/package.mk [hunk 2]
```diff
@@ -17,9 +17,9 @@
 ################################################################################
 
 PKG_NAME="game.libretro.dosbox"
-PKG_VERSION="c06c754"
-PKG_SHA256="75f711cd54f3f662673b9e1e2e51c31e99bcc7cb8d384b762a544f04a67239cf"
-PKG_REV="104"
+PKG_VERSION="b579067"
+PKG_SHA256="da5fafe44d571259ad22444b527d9a9ddea18fc90216b115abcd2d7b3ed7eebd"
+PKG_REV="105"
 PKG_ARCH="none"
```
Comment (matejmosko) [inline packages/mediacenter/kodi-binary-addons/game.libretro.dosbox/package.mk]: Is there any reason why dosbox is disabled in libreelec? (works on Ri)

File: packages/emulation/libretro-ppsspp/package.mk [hunk 3]
```diff
@@ -17,8 +17,8 @@
 ################################################################################
 
 PKG_NAME="libretro-ppsspp"
-PKG_VERSION="5f7bcf7"
-PKG_SHA256="09e61300c05705b1f98e1b575e44d366e5a243cc3be97b3a09ad420581459f87"
+PKG_VERSION="9145287"
```
Comment (CGarces) [inline packages/emulation/libretro-ppsspp/package.mk]: That gives me a 404. 
Is the file uploaded to sources?

Comment (CvH) [reply packages/emulation/libretro-ppsspp/package.mk]: is not yet uploaded, also not the previous (current) version, as `game.libretro.ppsspp` is not working at all it doesn't makes too much sense currently

**Comment by walterheisenberg** [comment]
Please update all packages from kodi-game
There are some packages missing, e.g. vice (C64), dosbox(DOS) or UAE (Amiga)
Forum thread:
https://forum.kodi.tv/showthread.php?tid=298461&pid=2700378#pid2700378
Thx

**Pull request merged**